### ISSUE TITLE
why can not find the downloadfile in android?

### ISSUE THREAD

**Issue by 1050509299** [johanneslumpe/react-native-fs#269]
print path like:/data/user/0/com.reactnativefstest/files/698.jpg,but the file does not exist

**Comment by 1050509299** [comment]
my project is https://github.com/cjdell/react-native-fs-test

**Comment by sedders123** [comment]
I found using `RNFS.ExternalDirectoryPath` instead of `RNFS.DocumentDirectoryPath` allowed my code to work

**Comment by CandyKTN** [comment]
@sedders123 right,but only use RNFS.ExternalStorageDirectoryPath that can find downloaded file in android,other path can't find  downloaded file

**Comment by my-name-is-nheo** [comment]
Go to system files and still cannot locate it. path looks similar as well @CandyKTN
